### PR TITLE
Testing that self.id don't work

### PR DESCRIPTION
**Pull request by eugeniyost**
# Testing that self.id don't work

Refs #774

### CODE REVIEW

File: tests/typecheck/fields/test_base.yml [hunk 1]
```diff
@@ -180,3 +180,11 @@
         obj = MyModel()
 
         reveal_type(obj.small) # N: Revealed type is "builtins.int*"
+
+
+-  case: test_access_to_id_field_through_self_if_no_primary_key_defined
+   main: |
+        from django.db import models
+        class MyModel(models.Model):
```
Comment (sobolevn) [inline tests/typecheck/fields/test_base.yml:188]: Since I've never had models with 0 fields in Django, what is going to happen in runtime? Does it work? 🤔

Comment (eugeniyost) [reply tests/typecheck/fields/test_base.yml:188]: This works, in this model there will be one field - id. But for reproduce error, it doesn't matter how many fields there will be in the model if the field with `primary_key=True` undefined.

Comment (sobolevn) [reply tests/typecheck/fields/test_base.yml:188]: So, why does this work then? 🤔 
https://github.com/typeddjango/django-stubs/pull/777/files#diff-1f9420e92c89c34537838f68941bff315ab1c4057d5940cfb065caf8ab40b062R22

**Comment by sobolevn** [comment]
@eugeniyost I've fixed the CI, can you please rebase your PR?

**Pull request closed without merging**